rhombus/rhombus/scribblings/ref-rx-pattern.scrbl

```diff
@@ -0,0 +1,664 @@
+#lang rhombus/scribble/manual
+@(import:
+    "common.rhm" open
+    rhombus/rx open
+    scribble/rx open
+    meta_label:
+      rhombus/rx open)
+
+@(def rx_eval = make_rhombus_eval())
+@examples(
+  ~eval: rx_eval
+  ~hidden:
+    import rhombus/rx open
+)
+
+@title(~tag: "rx-pattern"){Regexp Patterns}
+
+The portion of a @rhombus(rx) or @rhombus(rx_in) form within @quotes is
+a pattern that is written with regexp pattern operators. Some pattern
+operators overlap with expression operators, but they have different
+meanings and precedence in a pattern. For example, the pattern operator
+@rhombus(*, ~at rhombus/rx) creates a repetition pattern, instead of
+multiplying like the expression @rhombus(*) operator.
+
+@doc(
+  rx.macro '#%literal $string'
+  rx.macro '#%literal $bytes'
+){
+
+ A literal @tech{string} or @tech{byte string} can be used as a pattern.
+ It matches the string's characters or bytes literally. See also
+ @rhombus(case_insensitive, ~at rhombus/rx).
+
+@examples(
+  ~eval: rx_eval
+  rx'"hello"'.match("hello")
+  rx'"hello"'.match("olleh")
+  rx'#"a"'.match(#"a")
+)
+
+}
+
+
+@doc(
+  rx.macro '$pat #%juxtapose $pat'
+  rx.macro '$pat ++ $pat'
+  rx.macro '$pat #%call ($pat)'
```

Comment:
This seems like implementation details that readers shouldn't need to know. See also my comment below about "organizing via surface constructs"